packages/@aws-cdk/aws-lambda-event-sources/lib/index.ts

```diff
@@ -4,3 +4,4 @@ export * from './sqs';
 export * from './s3';
 export * from './sns';
 export * from './api';
+export * from './stream';
```

Comment:
This is an internal refactoring to reduce code duplication. I don't see value in exporting these to consumers (i.e., make them public API). In fact, it might limit our flexibility to change this in the future.